packages/commonwealth/server.ts

```diff
@@ -267,7 +268,10 @@ async function main() {
   const globalActivityCache = new GlobalActivityCache(models);
 
   // initialize async to avoid blocking startup
-  if (!NO_GLOBAL_ACTIVITY_CACHE) globalActivityCache.start();
+  if (!NO_GLOBAL_ACTIVITY_CACHE) {
+    globalActivityCache.start();
+    getChainActivity.startTask(models);
```

Comment:
@jnaviask is it ok to run this new query every 5mins to refresh and pre-fetch part of `/api/status` ?

Reply:
new Background task
<img width="1357" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/4791635/73f488f2-c0c2-4f07-b306-5f3ed666fb96">

Reply:
we can choose longer TTL now, we took care of refreshing results on changes t chain!
Thanks!

Reply:
removed background seems not required anymore.